Content.Shared/GameObjects/Components/PDA/UplinkCategory.cs

```diff
@@ -0,0 +1,8 @@
+namespace Content.Shared.GameObjects.Components.PDA
+{
+    public enum UplinkCategory
+    {
+        Weapon,
```

Comment:
Should probably be controlled by some kind of prototype (is what PJB will comment).

Reply:
I'm fine with either, honestly.